students/itsabhi/Lesson3/mailroom.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+
+
+def sending_thanks(d):
+    donor_hist = d
+    unique_donors = []
+    for entry in donor_hist:
+        name = entry[0]
+        if name not in unique_donors:
+            unique_donors.append(name)
+
+    donor = input("Type the Full Name of the donor here: ")
+    if donor == "list":
+        print(unique_donors)
+        donor = input("Type the Full Name of the donor here: ")
+    if donor not in unique_donors:
```

Comment:
I would consider using a `while` loop for your user input here so you don't have to repeat the prompts in your code